index.js

```diff
@@ -314,7 +316,7 @@ class Ky {
 			this._decorateResponse(response);
 
 			if (!response.ok && this._options.throwHttpErrors) {
-				throw new HTTPError(response);
+				throw new HTTPError(response, this.request, this._optoins);
```

Comment:
```suggestion
				throw new HTTPError(response, this.request, this. _options);
```

Reply:
Watch out, there's an added space there ☝️ 

```suggestion
				throw new HTTPError(response, this.request, this._options);
```

Reply:
Thanks for spotting @nielsdB97 👀!  I edited the suggestion